assets/scss/style.scss

```diff
@@ -213,3 +215,106 @@ html, body {
 section.comments {
   margin-top: 2.5rem;
 }
+
+/* === All styles related to the author page, /author/author-name. === */
+
+$img-dimension: 20rem;
+$author-padding: 6rem;
+
+.main-header {
+
+}
+
+/* author cover photo */
+.author-head {
+	background-repeat: no-repeat;
+	background-position: center center;
+	background-size: contain;
+
+	height: 30rem;
+	/* TODO - just do it based on margins*/
+
+
+}
+
+
+
+/* if author has no cover photo. */
+.no-cover {
+    height: 0em;
+}
+/* Container class for author photo, bio, location, and other meta data. */
+.author-profile {
+    text-align: center;
+
+    /*
+    padding-left: $author-padding;
+    padding-right: $author-padding;
+	*/
+    margin-bottom: -4rem; /*TODO play with this value*/
+
+}
+
+
+
+
+
+
+/* head shot*/
+.author-image {
+    width: $img-dimension;
+    margin: 0 auto;
+}
+
+.author-image .img {
+    margin: 0 auto;
+    height: $img-dimension;
+    background-size: cover;
+    background-position: center center;
+    border: 1px solid $primary-color;
```

Comment:
Hard to say. One one hand, definitely cleaner is better, on the other hand, if a user has a light image it's nice to have the delineation from the rest of the page. Your call.

No border: 
![no-border](https://cloud.githubusercontent.com/assets/5771615/6205689/106a3100-b52c-11e4-8a49-423c1c9cacef.png)

With border:
![yes-border](https://cloud.githubusercontent.com/assets/5771615/6205688/106778ca-b52c-11e4-9489-440d679381b9.png)